pom.xml

```diff
@@ -20,9 +20,10 @@
 
   <properties>
     <java.level>8</java.level>
-    <jenkins.version>2.303.3</jenkins.version>
+    <jenkins.version>2.346.1</jenkins.version>
     <hpi.compatibleSinceVersion>1.0.0</hpi.compatibleSinceVersion>
```

Comment:
Do you mind updating the reference to the minimum supported version in the readme as well? https://github.com/jenkinsci/datadog-plugin/blob/master/README.md#installation

Reply:
thanks, fixed

Reply:
Thanks!